src/theme/components/structuredContent.ts

```diff
@@ -0,0 +1,37 @@
+const StructuredContent = {
+  parts: ["image", "imageContainer"],
+  baseStyle: ({ imagePosition }) => {
+    const styles: { maxWidth?: string } = {};
+    if (imagePosition === "center") {
+      styles.maxWidth = "100%";
+    }
+    return {
+      imageContainer: {
+        float: imagePosition !== "center" ? imagePosition : undefined,
+        marginBottom: "m",
+        marginLeft:
+          imagePosition === "center"
+            ? null
+            : imagePosition === "left"
+            ? "0"
+            : "m",
+        marginRight:
+          imagePosition === "center"
+            ? null
+            : imagePosition === "left"
+            ? "m"
+            : "0",
+        width: imagePosition !== "center" ? "100%" : undefined,
+        ...styles,
+      },
+      image: {
```

Comment:
The `image` also needs `marginBottom: "m"` when the `imageContainer` is not used (i.e. no caption and no credit).  See the last example on the story page.

Reply:
This needs some more work.  I'll setup a call so we can quickly go over this together.